pkg/platform/provider/baremetal/constants/constants.go

```diff
@@ -133,4 +133,6 @@ const (
 	MinNumCPU = 2
 
 	APIServerHostName = "api.tke.com"
+
+	NeedUpgradeCoreDNSK8sVersion = "1.19.0"
```

Comment:
from 1.19,  we should support upgrade coredns to align with k8s,  so we need to make this process automatically and don't hard code coredns version in our hand.  all install and upgrade logic will handled by kubeadm